docs/modules/user-guide/partials/assembly_adding-commands-to-a-devfile.adoc

```diff
@@ -0,0 +1,24 @@
+ifdef::context[:parent-context-of-assembly_adding-commands-to-a-devfile: {context}]
+
+
+ifndef::context[]
+[id="assembly_adding-commands-to-a-devfile"]
+endif::[]
+ifdef::context[]
+[id="assembly_adding-commands-to-a-devfile_{context}"]
+endif::[]
+= Adding commands to a devfile
+
+[role="_abstract"]
+A devfile allows you to specify commands to run in a workspace. Every command can contain a subset of actions. These subsets are related to specific components and are run in the component containers.
```

Comment:
```suggestion
A devfile enables you to specify commands to run in a workspace. Every command can contain a subset of actions. These subsets are related to specific components and are run in the component containers.
```
Avoid giving inanimate things like software features human attributes. `Enable` can almost always replace `allow`, and `enable` almost always sounds more empowering for users. The one exception to this would be when you are documenting security features, and you're talking about allowing or permitting users to do certain things in the system.